### PR TITLE
Explicitely mention disabling validation through

### PR DESCRIPTION
**Pull request by HallerPatrick**
## What does this PR do?

This PR adds another code example for the `limit_val_batches` parameter description of the `Trainer` class. 
There is no explicit parameter to disable validation during training; this is achieved by setting `limit_val_batches=0.0`.
This is mentioned in the [source code](https://github.com/PyTorchLightning/pytorch-lightning/blob/29fe1dad155ee80e8934cf91004b4ea9cdbc455d/pytorch_lightning/trainer/trainer.py#L1866) through an error throw.

I, therefore, would add a quick, simple code example that addresses it.

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [x] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/common/trainer.rst [hunk 1]
```diff
@@ -871,6 +871,9 @@ Useful when debugging or testing something that happens at the end of an epoch.
     # run for only 10 batches
     trainer = Trainer(limit_val_batches=10)
 
+    # disable validation
+    trainer = Trainer(limit_val_batches=0.0)
```
Comment (akihironitta) [inline docs/source/common/trainer.rst:875]: This could be simpler.
```suggestion
    trainer = Trainer(limit_val_batches=0)
```

**Review by carmocca — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by Borda — APPROVED** [review]
